### PR TITLE
Readme remake 

### PR DESCRIPTION
**Pull request by auanasgheps**
Hi there,

I love this container and the project itself, you may also remember me for a couple of issues I've opened in the past.

I thought the README was a bit messy and unpolished, so I've remade most of it.

I hope you like it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,45 +17,83 @@ This image allows you to have JDownloader 2 daemon installed easily thanks to Do
...
 
 Appendixes
 ---
 
 ### Direct Connection
 
+To enable Direct Connection mode from internet, you need to forward the port 3129 in your Router. Please find more information in this [JDownloader's article](https://support.jdownloader.org/Knowledgebase/Article/View/33/0/myjdownloader-advanced-settings)
 As @jiaz83 stated
 
+**DNS Rebind Warning** if you are running a router like Fritz!Box, Asus, OpenWRT, DDWRT, pfSense or any other 3rd party *advanced* routers you may have DNS Rebind Protection enabled: Direct Connections will not work, you will have to explicitly whitelist `mydns.jdownloader.org`. The procedure is different for every router, here are some tips:
```
Comment (jaymoulin) [inline README.md:88]: I don't remember jiaz83 stating this. Could you please place the whole block after his quote?

Comment (auanasgheps) [reply README.md:87]: Fixed.

**Comment by auanasgheps** [comment]
I have done additional small fixes. Please let me know if there's anything else you want to improve.

Also if you are okay with it I could remove the quote and re-write the explanation for JDownloader direct connection.

**Comment by jaymoulin** [comment]
I already squash and merge your PR. Please open a new one if you want to make changes

**Pull request closed without merging**